### DISCUSSION TITLE
npm i next-cloudinary  ERROR npm i NEXT 14.0.0

### DISCUSSION THREAD

**Discussion by stcore** [cloudinary-community/next-cloudinary#331] [General]
E:\app>npm i next-cloudinary
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: erpnext@0.1.0
npm ERR! Found: next@14.0.0
npm ERR! node_modules/next
npm ERR!   next@"^14.0.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer next@"^12 || ^13" from next-cloudinary@4.28.0
npm ERR! node_modules/next-cloudinary
npm ERR!   next-cloudinary@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! 
npm ERR! For a full report see:
npm ERR! C:\Users\User\AppData\Local\npm-cache\_logs\2023-10-28T20_26_07_542Z-eresolve-report.txt

npm ERR! A complete log of this run can be found in: C:\Users\User\AppData\Local\npm-cache\_logs\2023-10-28T20_26_07_542Z-debug-0.log

**Comment by colbyfayock** [comment]
Duplicate https://github.com/colbyfayock/next-cloudinary/issues/328
